### PR TITLE
feat(pie-docs): DSW-1639 Add the chip code page

### PR DESCRIPTION
**Pull request by raoufswe**
## Describe your changes (can list changeset entries if preferable)
- adds the code section of the chip component

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code
- [x] If it is a `PIE Docs` change, I have reviewed the Docs site preview

### Reviewer 1
- [x] If it is a `PIE Docs` change, I have reviewed the PR preview

### Reviewer 2
- [x] If it is a `PIE Docs` change, I have reviewed the PR preview

### CODE REVIEW

File: apps/pie-docs/src/components/chip/code/slots.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+                "item": ["default"]
+            },
+            "The default slot is used to pass text into the chip component."
+        ],
+        [
+            {
+                "type": "code",
+                "item": ["icon"]
+            },
+            "Used to pass in an icon to the chip component."
```
Comment (xander-marjoram) [inline apps/pie-docs/src/components/chip/code/slots.json:16]: Nit
```suggestion
            "Used to pass an icon into the chip component."
```

File: apps/pie-docs/test/snapshots/expected-routes.snapshot.json [hunk 2]
```diff
@@ -62,10 +63,6 @@
     "components/tabs",
     "components/tag/code",
     "components/tag/overview",
-    "components/template/code",
```
Comment (xander-marjoram) [inline apps/pie-docs/test/snapshots/expected-routes.snapshot.json:65]: Did these need deleting?

File: packages/components/pie-chip/README.md [hunk 3]
```diff
@@ -36,81 +27,13 @@ $ npm i @justeattakeaway/pie-chip
...
-> When using the React version of the component, please make sure to also
-> include React as a [peer dependency](#peer-dependencies) in your project.
-
-
-## Peer Dependencies
+## Documentation
 
-> [!IMPORTANT]
-> When using `pie-chip`, you will also need to include a couple of dependencies to ensure the component renders as expected. See [the PIE Wiki](https://github.com/justeattakeaway/pie/wiki/Getting-started-with-PIE-Web-Components#expected-dependencies) for more information and how to include these in your application.
+Visit  [Link | PIE Design System](https://pie.design/components/chip/overview) to view more information on this component.
```
Comment (xander-marjoram) [inline packages/components/pie-chip/README.md:32]: I think "Link" isn't the most helpful text, and maybe we can give a hint about what information it includes, like usage, props and slots. That would also help anyone searching the page with Ctrl+F! 😄

Comment (raoufswe) [reply packages/components/pie-chip/README.md:32]: screams copy paste :P updated

File: apps/pie-docs/test/snapshots/expected-routes.snapshot.json (hunk 2 above)
Comment (raoufswe) [reply apps/pie-docs/test/snapshots/expected-routes.snapshot.json]: nope

**Review by LTurns — APPROVED** [review]

**Review by xander-marjoram — APPROVED** [review]

**Pull request merged**